rollup.config.js

```diff
@@ -35,7 +35,7 @@ export default {
     }),
     uglify({
       compress: {
-        warnings: false,
+        // warnings: false,
```

Comment:
Can you provide the exact yarn version used and the command run so I can test this? I did not have a problem with this locally.

Reply:
Hey @gabrielliwerant, I'm using yarn 1.15.2. The command I'm using is `yarn upgrade mui-datatables@https://github.com/shawnmitchell/mui-datatables.git#enhancement-on-table-init` so that grabs the fork from my repo and rebuilds locally. This is not part of the pull request per se, I just modified it so I could link to my stuff in my project.

Reply:
Thanks for that. Yeah, it looks like the warning option is deprecated in the `compress` config. I'll fix in a separate PR, so as not to tangle it with other issues.